tests/storages_tests/journal_tests/test_journal.py

```diff
@@ -34,26 +37,35 @@
 
 
 class JournalLogStorageSupplier:
-    def __init__(self, storage_type: str) -> None:
+    def __init__(self, storage_type: str, deprecated_classname: bool = False) -> None:
```

Comment:
I think we should not consider deprecated classes in this supplier, because those only should be tested as they are deprecated. How about just using newly introduced ones here?